src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -67,12 +67,72 @@ const convertEntityIdsToTags = (ids: string[], entities: CLM.EntityBase[]): OF.I
         .map<OF.ITag>(convertEntityToTag)
 }
 
-const getSuggestedTags = (filterText: string, allTags: OF.ITag[], tagsToExclude: OF.ITag[]): OF.ITag[] => {
+// Entities that can be chosen for required / blocking
+const conditionalEntityTags = (entities: CLM.EntityBase[]): IConditionalTag[] =>  {
+
+    // Ignore resolvers and negative entities
+    const filteredEntities = entities.filter(e => !e.doNotMemorize && !e.positiveId)
```

Comment:
I think this `doNotMemorize` term was added while I was gone. What does `doNotMemorize` mean? Or what's it used for?